mlflow/langchain/_langchain_autolog.py

```diff
@@ -121,8 +121,7 @@ def _inject_mlflow_callback(func_name, mlflow_callback, args, kwargs):
             config = RunnableConfig(callbacks=callbacks)
         else:
             callbacks = config.get("callbacks") or []
-            callbacks.append(mlflow_callback)
-            config["callbacks"] = callbacks
+            config["callbacks"] = [*callbacks, mlflow_callback]
```

Comment:
Can we put some source code reference that explains why does `*` operator is required and how it works for `CallbackManager` instance?

Reply:
Oh I missed the list conversion 😓